src/main/java/core/basesyntax/Operations.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public enum Operations {
```

Comment:
remove this enum, you don't need it, you don't need to store these values anywhere